### DISCUSSION TITLE
Activate barrel button binding on tip pressure

### DISCUSSION THREAD

**Discussion by bmpq** [OpenTabletDriver/OpenTabletDriver#1260] [Ideas]
In my old wacom medium one (current tablet is xp-pen deco 01v2) there was an option for pen side buttons to work as toggles on what to trigger when the pen goes down. So when you hold the pen side button nothing happens until the tip pressure goes above 0, and then instead of triggering tip click it triggers the assigned binding on the side button. Can it be done here in this driver? Right now the default action is to trigger the bindings immediately no matter the pressure.

**Comment by InfinityGhost** [comment]
So in short, activate binding on pressure rather than when the button is clicked?

**Reply by bmpq** [reply]
Yeah, pretty much

**Comment by gonX** [comment]
This is now possible with the pressure threshold slider.

**Reply by Aurailus** [reply]
Sorry for the necropost, but where are you seeing this? I don't see any pressure threshold slider in my UI.
![image](https://github.com/user-attachments/assets/286d14e4-40eb-40d3-8f72-5e7e39052d37)

**Reply by X9VoiD** [reply]
Tip threshold

**Reply by Aurailus** [reply]
Tip Threshold seems like it just adds a deadzone to when I press the pen down in general. I thought OP was looking for a way to make a side-button on the pen trigger when held down + pressing the pen tip onto the screen?

**Reply by X9VoiD** [reply]
So activating different bindings based on pressure, similar to a feature of some hall effect keyboards.

**Reply by Aurailus** [reply]
Sort of. Specifically, I want something to trigger when i both press down a pen button & apply pressure to the screen. e.g. holding the eraser key making the eraser active when i draw, not just when my pen is hovering over the canvas.
